automation/test.sh

```diff
@@ -363,7 +362,11 @@ if [[ -z ${KUBEVIRT_E2E_FOCUS} && -z ${KUBEVIRT_E2E_SKIP} ]]; then
   elif [[ $TARGET =~ (cnao|multus) ]]; then
     label_filter='(Multus,Networking,VMIlifecycle,Expose,Macvtap)'
   elif [[ $TARGET =~ sig-network ]]; then
-    label_filter='(sig-network)'
+    if [[ $TARGET =~ sig-network-istio ]]; then
```

Comment:
enough to check if istio is there, because you are under if sig-network already (and anyhow istio is enough to have)

Reply:
Fixed

> > /hold
> > This needs a patch on project-infra, introducing Istio lane.
> 
> Aint it Vice versa ? if you have the lane without something that strips the `-istio` the KUBEVIRT_PROVIDER wont be found (see please comment above)

You are right. Should be fixed in the next push